CaloMC/fcl/common.fcl

```diff
@@ -1,6 +1,6 @@
 BEGIN_PROLOG
   HitMakerDigitizationStart    : 450.
-  HitMakerDigitizationEnd      : 2005.0
+  HitMakerDigitizationEnd      : 1795
```

Comment:
Is it understood why this number extends beyond the microbunch?  Is it to include tails of showers that start near the end of the microbunch period?  Note that, in the MDC simulations, we only accumulate pileup 10ns past the end of the microbunch, so simulations of the times past that are not necessarily physically accurate.

Reply:
It needs to go over the microbunch length if you want to properly digitize a signal at the boundary of the microbunch. The exact number is a guess on my side, I'll fix it once the FPGA code is finalized. At this point, I'll take anything past the microbunch

Reply:
OK.  We will need to understand what this implies for simulated calo signals that extend past the microbunch end, my guess is the pileup for those will be underestimated, perhaps by a lot.  That issue is outside the scope of this PR.